vllm/model_executor/weight_utils.py

```diff
@@ -263,7 +263,8 @@ def hf_model_weights_iterator(
                 param = np.load(f)
             yield name, torch.from_numpy(param)
     elif load_format == "tensorizer":
-        logger.warning(
+        tensorizer_args = load_format.params
+        warnings.warn(
```

Comment:
Use a custom warning category here like `PerformanceWarning` from my previous comment to allow it to be easily filterable.